src/bms/command/minor_command.rs

```diff
@@ -10,6 +10,18 @@ use crate::bms::command::time::ObjTime;
 #[cfg_attr(feature = "serde", derive(serde::Serialize, serde::Deserialize))]
 pub struct ExWavPan(i64);
 
+impl AsRef<i64> for ExWavPan {
+    fn as_ref(&self) -> &i64 {
+        &self.0
+    }
+}
+
+impl From<ExWavPan> for i64 {
+    fn from(value: ExWavPan) -> Self {
+        value.0
+    }
+}
```

Comment:
`ExWavPan` and `ExWavVolume` should not impl `From` conversion from the internal type because they have own additional invariant about the contained.